aclmapping/oracle/mappings.go

```diff
@@ -55,15 +58,12 @@ func MsgVoteDependencyGenerator(keeper aclkeeper.Keeper, ctx sdk.Context, msg sd
 		// set exchange rate vote - WRITE
 		{
 			ResourceType:       sdkacltypes.ResourceType_KV_ORACLE_AGGREGATE_VOTES,
-			AccessType:         sdkacltypes.AccessType_READ,
-			IdentifierTemplate: msgVote.Validator,
+			AccessType:         sdkacltypes.AccessType_WRITE,
```

Comment:
This is a SET operation so it should have been a write (validation logic caught this)